test/e2e/e2e_suite_test.go

```diff
@@ -88,6 +90,10 @@ var (
 
 	// chartPath is the path to the operator chart.
 	chartPath string
+
+	// isolatedMode instructs the test to run without ngrok and exposing the cluster to the internet. This setup will only work with CAPD
+	// or other providers that run in the same network as the bootstrap cluster.
+	isolatedMode bool
```

Comment:
Ngrok ingress is provisioned in full if it has env set described in README. Isolated mode could be a default option, when the environment is not set.

Reply:
I'd like ngrok to be the default, when we run tests locally ngrok works for all setups. Using kind node IP will not work with docker desktop on mac :)